indico/modules/events/operations.py

```diff
@@ -384,3 +385,13 @@ def sort_reviewing_questions(questions, new_positions):
         field.position = index
     db.session.flush()
     logger.info('Reviewing questions of %r reordered by %r', questions[0].event, session.user)
+
+
+def create_event_request(event, category):
+    assert event.category != category
+    rq = EventMoveRequest(event=event, category=category, submitter=session.user)
+    db.session.flush()
+    logger.info('Category move request %s to %s created by %s', rq, category, session.user)
+    event.log(EventLogRealm.event, LogKind.change, 'Event', f'Move request to {category.title} created',
```

Comment:
In this log entry it would be useful to include the category chain in the detailed data (like we do in Event.move())

Reply:
Added the chain and ID.